.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -117,33 +123,9 @@ jobs:
           ref: refs/heads/stackhpc/${{ needs.generate-tag.outputs.openstack_release }}
           path: src/kayobe
 
-      # FIXME: Failed in kolla-ansible : Ensure the latest version of pip is installed
-      - name: Install dependencies
+      - name: Make sure dockerd is running and test Docker
```

Comment:
Is this always required or just for testing?

Reply:
It's not required, but has been useful for checking that the dind configuration for the runner is working. It seems to run very quickly - it's a tiny image